instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/sessionreplay/BaseSessionReplayActivity.kt

```diff
@@ -31,6 +31,8 @@ internal abstract class BaseSessionReplayActivity : AppCompatActivity() {
         val sessionReplayPrivacy = intent.getSessionReplayPrivacy()
         val sessionReplaySampleRate = intent.getSrSampleRate()
         Datadog.setVerbosity(Log.VERBOSE)
+        // make sure the previous instance is stopped
+        Datadog.stopInstance()
```

Comment:
not quite but there was some flakiness on my end and wanted to actually make sure that everything is stopped before my test starts to avoid any issues